src/templating/native/nativeTemplateEngine.js

```diff
@@ -10,7 +13,16 @@ ko.nativeTemplateEngine.prototype['renderTemplateSource'] = function (templateSo
         templateNodes = templateNodesFunc ? templateSource['nodes']() : null;
 
     if (templateNodes) {
```

Comment:
Could this be `if (templateNodes && templateNodes instanceof Array)` or just `if (templateNodes instanceof Array)`?